File: python/create_sender.py
import requests
import json

url = "<REPLACE_WITH_PROD_URL>/v1/senders/create"
payload = json.dumps({
  "name": "My Sender"
})
headers = {
  'Authorization': 'Bearer <REPLACE_WITH_YOUR_API_KEY>',
  'Content-Type': 'application/json'
}

response = requests.post(url, data=payload, headers=headers)
data = response.json()
print(json.dumps(data))

File: js/create_sender.js
const prodUrl = '<REPLACE_WITH_PROD_URL>'
const token = '<YOUR_TOKEN>'

const sendData = async () => {
  try {
    const data = JSON.stringify({
      name: 'My Sender',
    })

    const url = `${prodUrl}/v1/senders/create`

    const response = await fetch(url, {
      method: 'POST',
      headers: {
        Authorization: `Bearer ${token}`,
        'Content-Type': 'application/json',
      },
      body: data,
    })

    const responseData = await response.json()
    console.log(JSON.stringify(responseData))
  } catch (error) {
    console.log(error)
  }
}

sendData()
